website/docs/developer/research/research-logging-libraries.md

```diff
@@ -0,0 +1,142 @@
+---
+id: research-logging-libraries
+title: Logging Libraries Research Notes
+description: Research notes on some potential logging libraries.
+---
+
+## Preconditions
+- As ORCA runs in AWS, logger should tie into AWS structures.
+- Presently, ORCA uses the Python CumulusLogger.
```

Comment:
add a link to logger 
https://github.com/nasa/cumulus-message-adapter-python#logging-with-cumuluslogger

Reply:
Added